rest-service/manager_rest/rest/resources_v3_1/manager_config.py

```diff
@@ -49,3 +49,33 @@ def _authorization_config():
             'permissions': cfy_config.authorization_permissions
         }
         return authorization
+
+
+class ManagerRestConfig(SecuredResource):
+    @rest_decorators.exceptions_handled
+    @authorize('manager_rest_config_get')
+    def get(self):
+        """
+        Get the Manager's REST configuration
+        :return: Manager's REST configuration data
+        """
+        request_args = request.args.to_dict(flat=False)
+        current_app.logger.info('Retrieving Manager REST config. filter: {0}'.
+                                format(request_args))
+        result = dict()
+        filter_by = request_args.get('filter', [])
+        for key in filter_by:
+            try:
+                result[key] = config.instance.to_dict()[key]
+            except KeyError as ke:
+                current_app.logger.error(
+                    'KeyError thrown: {0}'.format(ke.message))
+                raise CommandExecutionException(command='manager.config.get',
+                                                error=ke.message,
```

Comment:
you'll be yelled at by python, which emits a warning that .message is deprecated
use just str(ke) i guess

Reply:
Fixed